src/shared/regions/defaultRegionProvider.ts

```diff
@@ -73,6 +74,15 @@ export class DefaultRegionProvider implements RegionProvider {
     }
 }
 
+export function makeEndpointsResourceFetcher(extensionContext: vscode.ExtensionContext): ResourceFetcher {
```

Comment:
gigantic nit: `make` ? most of our code-bases use `create` for this behaviour. For some reason `make` sounds weird to me. Like I say, massive nit, feel free to ignore.

Reply:
`makeXxx()` is the common convention in this codebase. I'm open to a separate, dedicated pass to change them all to `createXxx()` though.

Reply:
no no - consistent with the local code base is perfect.